application/src/main/kotlin/no/kartverket/matrikkel/bygning/application/models/kodelister/EgenregistreringKoder.kt

```diff
@@ -53,3 +53,14 @@ enum class OppvarmingKode(
     Sentralvarme("Sentralvarme", "Sentralvarme"),
     AnnenOppvarming("Annen oppvarming", "Annen oppvarming")
 }
+
+enum class KildematerialeKode(
+    override val presentasjonsnavn: String,
+    override val beskrivelse: String
+) : IKode {
+    Selvrapportert("Selvrapportert", "Når kilden ikke er dokumenterbar. Du velger dette når du selv gjør en vurdering  av hvilken løsning som er riktig. Dette alternative velges når kilden er feks den som opprinnelig oppførte bygningen, står i hyttebok"),
+    Salgsoppgave("Salgsoppgave", "Ved kjøp og salg av eiendom utarbeides flere rapporter og de har hatt ulikt benevnelse. Denne kilden velges hvis du har en salgsrapport, tilstandsrapport, boligsalgsrapport, takstrapport."),
+    Byggesaksdokumenter("Byggesaksdokumenter", "Saksdokumenter fra byggesaken som for eksempel en byggetillatelse (igansettingstillatelse, midlertidig brukstillatelse, ferdisattest)."),
+    Plantegninger("Plantegninger", "Utarbeidet av en profesjonell. Velges hvis det ikke er plantegninger i en byggesak, men utarebeidet i annen sammenheng som feks en seksjoneringssak eller rehabilitering eller andre arbeider som ikke krever søknad og tillatelse etter plan- og bygningsloven."),
+    AnnenDokumentasjon("AnnenDokumentasjon", "Når du har dokumentasjon som ikke passer i de andre alternativene, men det er mulig å dokumentere valget ditt, kan dette velges.")
```

Comment:
```suggestion
    AnnenDokumentasjon("Annen dokumentasjon", "Når du har dokumentasjon som ikke passer i de andre alternativene, men det er mulig å dokumentere valget ditt, kan dette velges.")
```

Er typisk sånn vi har gjort alle andre presentasjonsnavn